tests/modules/nf-core/pretextsnapshot/nextflow.config

```diff
@@ -0,0 +1,14 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    // all sequences of a map in one picture
+    withName: PRETEXTSNAPSHOT_FULL {
```

Comment:
Could you switch these to make sur ethey are in the same order as the tests (tripped me up at first thinking they were flipped :sweat_smile: :